pom.xml

```diff
@@ -1019,14 +1021,13 @@
         <module>client-flink/flink-1.14-shaded</module>
       </modules>
       <properties>
-        <flink.version>1.14.0</flink.version>
-        <flink.binary.version>1.14</flink.binary.version>
+        <flink.version>1.14.6</flink.version>
         <jackson.version>2.6.7</jackson.version>
         <jackson.databind.version>2.6.7.3</jackson.databind.version>
         <lz4-java.version>1.4.0</lz4-java.version>
-        <scala.version>2.12.15</scala.version>
-        <scala.binary.version>2.12</scala.binary.version>
         <zstd-jni.version>1.4.4-3</zstd-jni.version>
```

Comment:
if the Flink distribution does not provide these jars, why should overwrite them here?